23-02-2024 21:02:11.790Z;functions.bot_text_art;      ____     
23-02-2024 21:02:11.790Z;functions.bot_text_art;     [____]    
23-02-2024 21:02:11.790Z;functions.bot_text_art;   |=]()()[=|  
23-02-2024 21:02:11.790Z;functions.bot_text_art;     _\==/__    _____    ____     _        _____    ___   _  __ 
23-02-2024 21:02:11.790Z;functions.bot_text_art;   |__|   |_|  |  _  \  / ___|   | |      |_   _| / ___| | |/ / 
23-02-2024 21:02:11.790Z;functions.bot_text_art;   |_|_/\_|_|  | |  | | | |      | |        | |   | |    | ' /  
23-02-2024 21:02:11.790Z;functions.bot_text_art;   | | __ | |  | |  | | | |      | |        | |   | |    |  <   
23-02-2024 21:02:11.790Z;functions.bot_text_art;   |_|[  ]|_|  | |__| | | |____  | |____   _| |_  | |__  | . \  
23-02-2024 21:02:11.790Z;functions.bot_text_art;   \_|_||_|_/  |_____/   \_____| |______| |_____| \ ___| |_|\_\ 
23-02-2024 21:02:11.790Z;functions.bot_text_art;     |_||_|    
23-02-2024 21:02:11.790Z;functions.bot_text_art;     | ||_|_   
23-02-2024 21:02:11.790Z;functions.bot_text_art;   |___||___|  
23-02-2024 21:02:11.790Z;functions.bot_text_art;               
23-02-2024 21:02:11.790Z;functions.open_RM;  OPEN PROGRAM RM
23-02-2024 21:02:19.359Z;functions.show_exception_and_exit;  Failed to set focus.
Traceback (most recent call last):
  File "C:\Projetos_Remessas\Remessa_Pensao\main.py", line 12, in <module>
    loggin_RM()
  File "C:\Projetos_Remessas\Remessa_Pensao\SUBPROGRAMS\functions.py", line 79, in loggin_RM
    user_rm.send_hotkey(f'{user}')
  File "C:\Projetos_Remessas\Remessa_Pensao\venv\Lib\site-packages\clicknium\core\service\invokerservice.py", line 157, in wrapper
    raise exception;
    ^^^^^^^^^^^^^^^
clicknium.common.models.exceptions.WindowError: Failed to set focus.
